raiden/raidenwebui/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
Actually, it's not personal.. this config is default from all projects created with `@angular/cli ng` tool, but I will remove it.